app/src/main/java/uc/edu/klopfsea/groupscheduler/ui/main/MainFragment.kt

```diff
@@ -10,15 +10,14 @@ import uc.edu.klopfsea.groupscheduler.R
 
 class MainFragment : Fragment() {
 
-    companion object {
-        fun newInstance() = MainFragment()
-    }
-
     private lateinit var viewModel: MainViewModel
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                               savedInstanceState: Bundle?): View {
         return inflater.inflate(R.layout.main_fragment, container, false)
     }
+    companion object {
```

Comment:
companion objects are supposed to come last in the class and not at the top according to the Kotlin coding conventions so i had moved your guys companion object to the bottom as opposed to being at the top as you guys had it at the beggining.